js/templates/modals/listingDetail/listing.html

```diff
@@ -2,8 +2,19 @@
 
 <div class="modalContentInner">
   <div class="listingControls flex">
+    <div class="jsModalClose contentBox clrP clrBr clrSh3" style="height: 35px; line-height: 35px; padding: 0 8px; cursor: pointer">
+      <div class="userIcon floL discTn clrBr2 clrSh1"
+        <% var avatarHash = ob.avatarHashes ? ob.isHiRez() ? ob.avatarHashes.small : ob.avatarHashes.tiny : ''; %>
+        <% if (avatarHash) { %>
+          style="background-image: url(<%= ob.getServerUrl(`ipfs/${avatarHash}`) %>), url('../imgs/defaultAvatar.png')"
+        <% } else { %>
+         style="background-image: url('../imgs/defaultAvatar.png');"
+        <% } %>>
+      </div>
+      <div style="overflow: auto; font-size: 13px">Store Name <u style="font-size: 12px">View store</u></div>
+    </div>
     <% if (ob.ownListing) { %>
-    <div class="btnStrip clrSh3">
+    <div class="btnStrip clrSh3" style="margin-left: auto; margin-right: 40px">
```

Comment:
Instead of putting an inline margin here, the listingControls class should have a rule to add space to the right if the close button is present. 

Then these buttons can be aligned to the right either with a flex alignment class, or the previous sibling of the btnStrip class having a flexExpand class.